Automattic/wp-calypso
# Add a promotion of support sessions to the NPS popup

#### Changes proposed in this Pull Request

* The NPS popup shows a promotion of support sessions in the final page when the user is eligible.

See p9jf6J-1L7-p2 for more detail.

#### Testing instructions



* Visit the Reader page as a signed user and you see the NPS popup.
* Run `npsSurvey();` in your browser console to open the NPS survey.
* If any of your sites are on the Business plan and you submit the score < 7, the NPS popup shows you a promotion of support sessions as follows in the final page. The message can change depending on how many available support sessions you currently have.
  * If you open the popup with `npsSurvey(true);`, it thinks you're available for support sessions.

_When you have available support sessions_
<img width="400" alt="Stats_‹_Taggon_on_wordpress_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/58718706-109ff000-8409-11e9-961b-ef7adf0f99e4.png">

_When you use up all support sessions_
<img width="400" alt="Stats_‹_Taggon_on_wordpress_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/58718838-50ff6e00-8409-11e9-9521-0695e06eef09.png">
